### PR TITLE
Dbeaver/pro#2522 app launcher 2

### PR DESCRIPTION
**Pull request by serge-rider**

### CODE REVIEW

**Review by E1izabeth — APPROVED** [review]

File: plugins/org.jkiss.dbeaver.launcher/src/org/jkiss/dbeaver/launcher/DBeaverLauncher.java [hunk 1]
```diff
@@ -1364,10 +1333,10 @@ private String computeDefaultUserAreaLocation(String pathAppendage) {
             try (FileInputStream inStream = new FileInputStream(eclipseProduct)) {
                 props.load(inStream);
                 String appId = props.getProperty(PRODUCT_SITE_ID);
-                if (appId == null || appId.trim().length() == 0)
+                if (appId == null || appId.trim().isEmpty())
```
Comment (ShadelessFox) [inline plugins/org.jkiss.dbeaver.launcher/src/org/jkiss/dbeaver/launcher/DBeaverLauncher.java:1336]:
```suggestion
                if (appId == null || appId.isBlank())
```

File: plugins/org.jkiss.dbeaver.launcher/src/org/jkiss/dbeaver/launcher/DBeaverLauncher.java [hunk 2]
```diff
@@ -1364,10 +1333,10 @@ private String computeDefaultUserAreaLocation(String pathAppendage) {
...
                     appId = ECLIPSE;
                 String appVersion = props.getProperty(PRODUCT_SITE_VERSION);
-                if (appVersion == null || appVersion.trim().length() == 0)
+                if (appVersion == null || appVersion.trim().isEmpty())
```
Comment (ShadelessFox) [inline plugins/org.jkiss.dbeaver.launcher/src/org/jkiss/dbeaver/launcher/DBeaverLauncher.java:1339]:
```suggestion
                if (appVersion == null || appVersion.isBlank())
```

File: plugins/org.jkiss.dbeaver.launcher/src/org/jkiss/dbeaver/launcher/DBeaverLauncher.java [hunk 3]
```diff
@@ -1879,11 +1848,11 @@ private String getProductProperties() throws IOException {
             try (FileInputStream inStream = new FileInputStream(eclipseProduct)) {
                 props.load(inStream);
                 String appId = props.getProperty(PRODUCT_SITE_ID);
-                if (appId == null || appId.trim().length() == 0) {
+                if (appId == null || appId.trim().isEmpty()) {
```
Comment (ShadelessFox) [inline plugins/org.jkiss.dbeaver.launcher/src/org/jkiss/dbeaver/launcher/DBeaverLauncher.java:1851]:
```suggestion
                if (appId == null || appId.isBlank()) {
```

File: plugins/org.jkiss.dbeaver.launcher/src/org/jkiss/dbeaver/launcher/DBeaverLauncher.java [hunk 4]
```diff
@@ -1879,11 +1848,11 @@ private String getProductProperties() throws IOException {
...
                     appId = ECLIPSE;
                 }
                 String appVersion = props.getProperty(PRODUCT_SITE_VERSION);
-                if (appVersion == null || appVersion.trim().length() == 0) {
+                if (appVersion == null || appVersion.trim().isEmpty()) {
```
Comment (ShadelessFox) [inline plugins/org.jkiss.dbeaver.launcher/src/org/jkiss/dbeaver/launcher/DBeaverLauncher.java:1855]:
```suggestion
                if (appVersion == null || appVersion.isBlank()) {
```

File: plugins/org.jkiss.dbeaver.launcher/src/org/jkiss/dbeaver/launcher/DBeaverLauncher.java (hunk 2 above)
Comment (serge-rider) [reply plugins/org.jkiss.dbeaver.launcher/src/org/jkiss/dbeaver/launcher/DBeaverLauncher.java:1339]: Agreed. But this code was copy-pasted from Eclipse launcher, we don't want to fix code style yet.
Probably we'll need to delete 70% of this code eventually.

**Review by ShadelessFox — APPROVED** [review]

**Pull request merged**